mantidproject/mantid
# Indirect Elwin - Obtain Units from Sample Logs

The elastic window multiple now reads the sample environment units from the sample logs instead of just assuming that the sample environment is in temperature.

**To test:**

`\\olympic\babylon5\Public\Louise McCann\DataAnalysis\Elwin`

Interfaces>Data Analysis>Elwin

Input Files: `osiris120225_graphite002_red`, `osiris120226_graphite002_red`, `osiris120227_graphite002_red`

SE log name: `Field`
Press `Run`
Once algorithm complete, plot workspace with the suffix `_elf`
The x-axis should be labeled `Field` and have units of Tesla (T)

Repeat with SE log name: `sample`
x-axis units should be in temperature (K)

Fixes #16019.

[Release Notes](https://github.com/mantidproject/mantid/blob/2b6eb15cb7dd9bbc22efcc02aacc444793036761/docs/source/release/v3.9.0/indirect_inelastic.rst)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
